test/e2e/tests/01-apply-recommendations/03-assert.yaml

```
@@ -0,0 +1,37 @@
+  labels:
+    app: busybox
+spec:
+  containers:
+  - name: busybox
+    resources:
+      requests:
+        cpu: "113m"
+        memory: "34M"
+status:
+  phase: Running
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)